static/app/gettingStartedDocs/java/java.tsx

```diff
@@ -36,6 +36,24 @@ export const steps = ({
     type: StepType.INSTALL,
     description: t('Install the SDK via Gradle, Maven, or SBT:'),
     configurations: [
+      {
+        description: (
+          <p>
+            {tct(
+              'To see source context in Sentry, you have to generate an auth token by visiting the [link:Organization Auth Tokens] settings. You can then set the token as an environment variable that is used by the build plugins.',
+              {
+                link: (
+                  <ExternalLink href="https://sentry.io/orgredirect/organizations/:orgslug/settings/auth-tokens/" />
```

Comment:
@ArthurKnaus can you please take a look at this link (it's duplicated across all platforms). I'm unsure what this should look like.

Reply:
Is there any reason why we need to use this redirect link?
If not I would have just redirected without leaving the app:
```
<Link to="/settings/auth-tokens/" />
```